app/hotels/src/filter/stars/StarsCheckbox.js

```diff
@@ -1,69 +1,34 @@
 // @flow
 
 import * as React from 'react';
-import { View, Text, StyleSheet, TouchableWithoutFeedback } from 'react-native';
-import { Color, Stars, Icon } from '@kiwicom/react-native-app-common';
+import { Text, StyleSheet } from 'react-native';
+import { Color, Stars, Checkbox } from '@kiwicom/react-native-app-common';
 
 type Props = {|
   onPress: () => void,
-  stars?: number,
+  stars: number,
   isChecked?: boolean,
-  text?: string,
   // $FlowFixMeProps
   style?: Object,
 |};
 
 export default function StarsCheckbox(props: Props) {
   return (
-    <TouchableWithoutFeedback onPress={props.onPress}>
-      <View style={[styles.wrapper, props.style]}>
-        <View style={styles.label}>
-          {props.stars ? (
-            <Text style={styles.stars}>
-              <Stars rating={props.stars} />
-            </Text>
-          ) : (
-            <Text style={styles.text}>{props.text}</Text>
-          )}
-        </View>
-        {props.isChecked && (
-          <View style={styles.check}>
-            <Icon name="check" size={26} color={Color.brand} />
-          </View>
-        )}
-      </View>
-    </TouchableWithoutFeedback>
+    <Checkbox
```

Comment:
This is the right direction how to compose and reuse components... 👍 (not the other way around) 😊